### ISSUE TITLE
PaymentRequest does not seem to set ShopperInteraction correctly

### ISSUE THREAD

**Issue by LinusCenterstrom** [Adyen/adyen-dotnet-api-library#7]
Doing the following

``` csharp
var p = new Payment(client);      
p.Authorise(new PaymentRequest
{
	SelectedRecurringDetailReference = "LATEST",
	Reference = Guid.NewGuid().ToString(),
	MerchantAccount = "HIDDEN",
	ShopperReference = "HIDDEN",
	Amount = new Amount("SEK", 5000),
	ShopperInteraction = ShopperInteractionEnum.Contauth,
	Recurring = new Adyen.EcommLibrary.Model.Reccuring.Recurring(Contract: Adyen.EcommLibrary.Model.Enum.Contract.Recurring),
});
```

I get an error 807 Invalid combination of shopper interaction and recurring-contract
where it looks like it received shopperinteraction Ecommerce (or most likely none at all)

However it works fine when I manually create the JSON myself with
``` csharp
var auth = new Authorise(new Payment(client))
auth.Request(JsonConvert.SerializeObject(
	new
	{
		selectedRecurringDetailReference = "LATEST",
		reference = Guid.NewGuid().ToString(),
		merchantAccount = "HIDDEN",
		shopperReference = "HIDDEN",
		amount = new
		{
			currency = "SEK",
			value = 5000
		},
		shopperInteraction = "ContAuth",
		recurring = new
		{
			contract = "RECURRING"
		}
	}
))
```
which leads me to believe something is wrong with how the PaymentRequest class handles
the serialization of ShopperInteraction

Env is Test btw if that matters

**Comment by AlexandrosMor** [comment]
Hello @LinusCenterstrom,

We fixed the issue in branch [here](https://github.com/Adyen/adyen-dotnet-api-library/tree/issue-7) You can try it and please let us know for the results. 

Kind regards, 
Alexandros

**Comment by LinusCenterstrom** [comment]
Alright, thanks for the quick fix, I'll try it out when its merged and a new Nuget is uploaded

**Comment by AlexandrosMor** [comment]
Hello @LinusCenterstrom, 
It is released in nuget version 1.0.4.
Install-Package Adyen.EcommLibrary -Version 1.0.4 
Regards
Alexandros

**Comment by subnormalseries** [comment]
I am not sure this works if the ShopperInteraction is set to Moto. I have created a pull request to resolve this.
